docs/source/monitoring.md

```diff
@@ -4,22 +4,36 @@ Both Prometheus, Grafana and AlertManager were configured with specific rules fo
 All of them will be available under the `scylla-monitoring` namespace.
 Customization can be done in `examples/common/monitoring/values.yaml`
 
-1. Add monitoring stack charts repository
+1. Download Scylla Monitoring
+
+   First you need to download Scylla Monitoring, which contains Grafana dashboards and custom Prometheus rules.
+   You can do this by running the following command:
+    ```
+    wget https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-4.0.0.tar.gz
+    tar -xvf scylla-monitoring-4.0.0.tar.gz
+    ```
+
+2. Add monitoring stack charts repository
```

Comment:
`1.` - it's best to let the renderer generate the numbers so the steps can be edited without having to adjust he whole file which would create unnecessary conflicts